src/DataTable/DataTable.svelte

```diff
@@ -100,6 +100,12 @@
   /** Set to `true` to use static width */
   export let useStaticWidth = false;
 
+  /** Set to `number` to limit the output */
```

Comment:
Let's make the `pageSize` description consistent with that from `Pagination.svelte`.

```suggestion
  /** Specify the number of items to display in a page */
```